packages/create-block/lib/templates/block/view.js.mustache

```diff
@@ -3,7 +3,7 @@
  * on posts/pages that contain this block.
  *
  * When this file is defined as the value of the `viewScript` property
- * in `block.json` it will be enqueued when, and only when, viewing the
+ * in `block.json` it will be enqueued on the front end of the site.
  * content on the front of the site - i.e. it will *not* run in the editor.
```

Comment:
I think we need to remove this part too?
```suggestion
 
```

Reply:
I'm not sure where you're seeing that. 🤷‍♂️